examples/Classification_new/main_predict_bert.py

```diff
@@ -0,0 +1,72 @@
+# Copyright 2020 The Forte Authors. All Rights Reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#      http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+"""This file predict the ner tag for conll03 dataset."""
+
+import yaml
+import torch
+from forte.pipeline import Pipeline
+from forte.predictor import Predictor
+from ft.onto.base_ontology import Sentence
+from forte.data.readers.imdb_reader import IMDBReader
+
+def pad_each_bach(word, max_sen_len):
```

Comment:
This `pad_each_bach` appears multiple times, maybe just create a util function for it?

And it is `pad_each_batch` instead.